example_code/applications/time_lock/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "stylus-timelock-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+alloy-primitives = "0.7.5"
+alloy-sol-types = "0.7.5"
+mini-alloc = "0.4.2"
+stylus-sdk = { version = "0.5.1" , features = ["docs"] }
```

Comment:
I can't seem to be able to build (`cargo stylus check`) with this configuration. I get the following error:

```
error[E0405]: cannot find trait `IntoLogData` in module `alloy_sol_types::private`
  --> src/lib.rs:19:1
   |
19 | / sol!{
20 | |     error AlreadyInitialized();
21 | |     error NotOwnerError();
22 | |     error AlreadyQueuedError(bytes32 txId);
...  |
45 | |     event Cancel(bytes32 indexed txId);
46 | | }
   | |_^ not found in `alloy_sol_types::private`
```